app/Http/Controllers/FileOtpController.php

```
@@ -0,0 +1,486 @@
+    /**
+     * Generate access token for verified OTP
+     */
+    private function generateAccessToken($otpSecurity): string
+    {
+        return base64_encode(json_encode([
+            'otp_security_id' => $otpSecurity->id,
+            'user_id' => $otpSecurity->user_id,
+            'expires_at' => now()->addMinutes(30)->timestamp,
+            'signature' => hash_hmac('sha256', $otpSecurity->id . $otpSecurity->user_id, config('app.key'))
+        ]));
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'generateAccessToken'. (undefined)

(UnusedPrivateMethod)